docker/daemon/daemon_types.go

```diff
@@ -41,7 +41,7 @@ type schema2Manifest struct {
 
 // Based on github.com/docker/docker/image/image.go
 // MOST CONTENT OMITTED AS UNNECESSARY
-type image struct {
+type dockerImage struct {
```

Comment:
This is only necessary to make it possible to import `containers/image/image`, as https://github.com/containers/image/pull/165/commits/29838ec474320414ed0dc02e166812a898d3e65e says.